home_assistant_glow.yaml

```diff
@@ -12,6 +12,9 @@ substitutions:
   status_led: GPIO5
   # imp/kWh rate ⬇ #
   pulse_rate: '1000'
+  
+dashboard_import:
+  package_import_url: github://klaasnicolaas/home-assistant-glow/home_assistant_glow.yaml
```

Comment:
Should we point this to a tag version?

Reply:
You can use a tag but you will have to maintain the tag until a hardware change comes i would ignore the tag, and 
if we get an ESP32 version we can then create a ...glow_32.yaml